deploy/charts/istio-csr/values.yaml

```
@@ -89,6 +89,8 @@ app:
     # 1 hour.
     istiodCertificateDuration: 1h
     istiodCertificateRenewBefore: 30m
+    # Create the default certificate as part of install.
+    issueDefaultCertificate: true
```

For some consistency :slightly_smiling_face: 

```suggestion
    istiodCertificateEnable: true
```